### PR TITLE
fix: delete the refresh in the update:executing: as it "rollbacks"

### PR DESCRIPTION
**Pull request by jvanecek**
… the said update

### CODE REVIEW

**Review by gcotelli — DISMISSED** [review]
LGTM!
We just need to fix the build. Seems that some changes in travis are breaking the RDBMS installation step :(

**Comment by serpi90** [comment]
> LGTM!
> We just need to fix the build. Seems that some changes in travis are breaking the RDBMS installation step :(

The problem is with the installation script, running it with docker should solve the problem.
I understand the change require to add docker to .travis.yml
```yml
services:
  - docker
```
then on the script start the corresponding pg version
`docker run -d -p 127.0.0.1:5432:5432 -e POSTGRES_PASSWORD=password posgres:9.4`
[Postgres on Docker Hub](https://hub.docker.com/_/postgres)

**Review by serpi90 — DISMISSED** [review]

File: source/Sagan-RDBMS/RDBMSRepository.class.st [hunk 1]
```diff
@@ -145,34 +139,42 @@ RDBMSRepository >> matchingCriteriaBuilder [
...
-	^ self withReadSessionDo: [ :session | session refresh: aMutableDomainObject ]
+	| updatedInstance |
+
+	self
+		transact: [ :session | 
+			updatedInstance := session
+				modify: aMutableDomainObject
+				in: [ aMonadycBlock value: aMutableDomainObject ]
+			].
+	^ updatedInstance
```
Comment (gcotelli) [inline source/Sagan-RDBMS/RDBMSRepository.class.st]:
```suggestion
	^ aMutableDomainObject
```

File: source/Sagan-RDBMS/RDBMSRepository.class.st [hunk 2]
```diff
@@ -145,34 +139,42 @@ RDBMSRepository >> matchingCriteriaBuilder [
...
 { #category : #management }
 RDBMSRepository >> update: aMutableDomainObject executing: aMonadycBlock [
 
-	self inUnitOfWorkDo: [ :session | session modify: aMutableDomainObject in: [ aMonadycBlock value: aMutableDomainObject ] ].
-	^ self withReadSessionDo: [ :session | session refresh: aMutableDomainObject ]
+	| updatedInstance |
+
+	self
+		transact: [ :session | 
+			updatedInstance := session
```
Comment (gcotelli) [inline source/Sagan-RDBMS/RDBMSRepository.class.st]:
```suggestion
			session
```

File: source/Sagan-RDBMS/RDBMSRepository.class.st [hunk 3]
```diff
@@ -145,34 +139,42 @@ RDBMSRepository >> matchingCriteriaBuilder [
...
+	^ self
+		withSessionDo: [ :session | self handleWriteErrorsDuring: [ session inUnitOfWorkDo: aBlock ] ]
 ]
 
 { #category : #management }
 RDBMSRepository >> update: aMutableDomainObject executing: aMonadycBlock [
 
-	self inUnitOfWorkDo: [ :session | session modify: aMutableDomainObject in: [ aMonadycBlock value: aMutableDomainObject ] ].
-	^ self withReadSessionDo: [ :session | session refresh: aMutableDomainObject ]
+	| updatedInstance |
```
Comment (gcotelli) [inline source/Sagan-RDBMS/RDBMSRepository.class.st]:
```suggestion 
```

File: source/Sagan-RDBMS/RDBMSRepository.class.st (hunk 1 above)
Comment (gcotelli) [reply source/Sagan-RDBMS/RDBMSRepository.class.st]: It's really necessary to have the temporal variable? `aMutableDomainObject` must be the object we want

**Review by gcotelli — CHANGES_REQUESTED** [review]

File: source/Sagan-RDBMS/RDBMSRepository.class.st (hunk 1 above)
Comment (jvanecek) [reply source/Sagan-RDBMS/RDBMSRepository.class.st]: Indeed, just fixed it

**Review by gcotelli — APPROVED** [review]

**Pull request merged**